internal/middleware/account_test.go

```diff
@@ -59,7 +58,56 @@ func TestAccountMiddleware(t *testing.T) {
 			assert.Equal(t, "124", acc.OrgID)
 		})
 
-		handler := middleware.AccountMiddleware(isAccInNext)
+		handler := AccountMiddleware(isAccInNext)
+		handler.ServeHTTP(rr, req)
+
+		count, err := stubs.AccountStubCount(ctx)
+		assert.Nil(t, err, fmt.Sprintf("Error creating a new request: %v", err))
```

Comment:
Why are we adding these lines? I do not think it is correct

Reply:
You are right, these should not be here.